src/components/label-info/LabelInfo.tsx

```diff
@@ -30,7 +30,7 @@ export const LabelInfo = ({label}: LabelInfoProps) => {
 
 const useStyles = StyleSheet.createThemeHook((theme) => ({
   flag: {
-    backgroundColor: theme.static.status.info.background,
+    backgroundColor: theme.static.background.background_accent_3.background,
```

Comment:
The color in Figma:
![Skjermbilde 2024-05-23 kl  10 29 05](https://github.com/AtB-AS/mittatb-app/assets/134292729/f02dfddf-b1e0-40e2-a817-fee5aae09d3c)
Is there a mismatch between Figma and design system?

Reply:
Yes, there's a bigger design system change that might need to be done soon:

https://github.com/AtB-AS/kundevendt/issues/15379
https://github.com/AtB-AS/kundevendt/issues/18138

I have a commit ready on design system, but I'm waiting until this Flexible ticket is done first, and then I can work on the design system issue. 

There's a type change for status icons, where the new design have Primary & Secondary colors instead of a single color, and we will need to change the usage after updating the new design system. 

Since I'm not sure if any of the OMS-partners wants to update the design system (Troms might need to update some illustrations, for example), I don't want to make the change until I can update the usage on the app repo.

Reply:
I see. Perhaps it's best to hold back the color change until DS is ready?
Otherwise we should also check how this looks in the other OMS apps before merging

Reply:
I think that is a good idea, I'll discuss with Morten when I start working on the design system :)

Reply:
Reverted